src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.targets

```diff
@@ -1,8 +1,7 @@
 <Project>
   <Import Project="Sdk.targets" Sdk="Microsoft.Android.Sdk"
       Condition=" '$(TargetPlatformIdentifier)' == 'android' " />
-  <Import Project="Sdk.targets" Sdk="Microsoft.Android.Sdk.BundleTool"
-      Condition=" '$(AndroidPackageFormat)' == 'aab' " />
+  <Import Project="Sdk.targets" Sdk="Microsoft.Android.Sdk.BundleTool" />
```

Comment:
Sorry, I guess we still need to check `Condition=" '$(TargetPlatformIdentifier)' == 'android' "` here. It should match the line above.

Otherwise we'd import this for a .NET 6 console app.